CONTRIBUTING.md

```diff
@@ -54,6 +57,27 @@ approach is to keep the *one step = one commit* approach.
 Avoid merge commits in pull requests - use `git rebase` to get rid of them. This helps us keep git
 tree clean and helps to make it easy to understand historical changes done in past commits.
 
+
+### Maintainer Information
+
+Reviewers should pay attention to the following points:
+
+The PR ...
+* ... has been approved by an SME of the area that the PR addresses, which indicates that it is generally a step in a right direction.
+* ... passes PR gating tests, or failing tests are waived in a comment by the reviewer explaining the reasoning.
+* ... adheres to the [coding style](docs/manual/developer/04_style_guide.md#complianceascode-style-guide).
+* ... has been tested. The following options that allow for testing various aspects of the project:
+  * [SSG Test Suite](tests/README.md) for rule tests to ensure that OVAL checks are correct and test the ability of remediations to satisfy those checks.
+  * [BATS framework](tests/unit/bash) for bash tests that allows for fast and exhaustive testing of remediations that are parametrized by Jinja2.
+  * [Unit tests](tests/unit) that test components of the build system as well as components of the SSG Test Suite.
+  * Ad-hoc tests that are integrated into the `ctest` chain directly, i.e. the shellcheck test.
+* ... updates READMEs, man pages or other documentation when changes of described behavior are introduced.
+* ... doesn't contain merge commits - those can be removed by rebasing.
+* ... is rebased against recent-enough branch that executes the whole range of expected integrated gating tests.
```

Comment:
nit: Maybe `relevant` versus `expected`? The expected behavior, IMO, would be for GitHub to auto-rebase the present target branch state and the proposed PR changes and then run CI on top of that, but sadly it doesn't and instead takes the base branch as-is. :-) Saying relevant would allow for some variation in main branch, but suggest that any tests which impact the proposed changes should be pulled in first. Thoughts?